src/WebRTCSession.h

```diff
@@ -5,15 +5,23 @@
 
 #include <QObject>
 
-#include "CallDevices.h"
 #include "mtx/events/voip.hpp"
 
 typedef struct _GstElement GstElement;
+class CallDevices;
 class QQuickItem;
 
 namespace webrtc {
 Q_NAMESPACE
 
+enum class CallType
+{
+        VOICE,
+        VIDEO,
+        SCREEN // localUser is sharing screen
```

Comment:
I feel like ideally we would support picture in picture during screenshare at some point, where you can send camera and screen contents at the same time. This may make that harder in the future, not sure?

Reply:
Ha I actually implemented this. Works fine, you just have to use a compositor like we do already. I didn't have the patience though to finish the details around sizing and positioning of the inset picture before PR'ing. Sometimes you just have to PR. I'm thinking the inset picture goes bottom right. Or just offer it as a user choice I suppose.

Reply:
Oh, yeah, if you have a plan for how to do it later, that is totally fine. I was just worried you may need to walk back these changes again. But I trust your judgement here :3